README.md

```diff
@@ -1,160 +1,125 @@
-  
-| Not Required | Required  |
-| - | - |
-| [Achievements](https://dev.epicgames.com/docs/game-services/achievements) | [Authentication](https://dev.epicgames.com/docs/epic-account-services/auth-interface) |
-| [Leaderboards](https://dev.epicgames.com/docs/game-services/leaderboards) | [Ecommerce](https://dev.epicgames.com/docs/epic-games-store/tech-features-config/ecom) |
-| [Lobby](https://dev.epicgames.com/docs/game-services/lobbies) | [Friends](https://dev.epicgames.com/docs/epic-account-services/eos-friends-interface) |
-| [Lobby with Voice](https://dev.epicgames.com/docs/game-services/voice#voicewithlobbies) | [Presence](https://dev.epicgames.com/docs/epic-account-services/eos-presence-interface) |
-| [NAT P2P](https://dev.epicgames.com/docs/game-services/p-2-p) | [Social Overlay](https://dev.epicgames.com/docs/epic-account-services/social-overlay-overview) |
-| [Player Data Storage](https://dev.epicgames.com/docs/game-services/player-data-storage) | [User Info Interface](https://dev.epicgames.com/docs/epic-account-services/eos-user-info-interface) |
-| [Sessions](https://dev.epicgames.com/docs/game-services/sessions) | [EOS Mod SDK](https://dev.epicgames.com/docs/epic-games-store/tech-features-config/mods) (Unsupported) |
-| [Stats](https://dev.epicgames.com/docs/game-services/eos-stats-interface) | [Progression Snapshot Interface](https://dev.epicgames.com/docs/epic-account-services/progression-snapshot) (Unsupported) |
-| [Title Storage](https://dev.epicgames.com/docs/game-services/title-storage) |  |
-| [Reports](https://dev.epicgames.com/docs/game-services/reports-interface) |  |
-| [Sanctions](https://dev.epicgames.com/docs/game-services/sanctions-interface) |  |
-| [Anti-Cheat](https://dev.epicgames.com/docs/game-services/anti-cheat) |  |
-| [Custom Invites](https://dev.epicgames.com/docs/game-services/custom-invites-interface) |  |
-| [Metrics](https://dev.epicgames.com/docs/game-services/eos-metrics-interface) |  |
-| [Connect Interface](https://dev.epicgames.com/docs/game-services/eos-connect-interface?sessionInvalidated=true) |  |
-| [Logging Interface](https://dev.epicgames.com/docs/game-services/eos-logging-interface) |  |
-| [Platform Interface](https://dev.epicgames.com/docs/game-services/eos-platform-interface) |  |
-| [Voice Trusted Server](https://dev.epicgames.com/docs/game-services/voice#voicewithatrustedserverapplication) (Unsupported) |  |
-</details>
-
-<br />
-
-# <div align="center">Getting Started</div> <a name="getting-started" />
+* Targets [EOS SDK 1.16.0](https://dev.epicgames.com/docs/epic-online-services/release-notes#116---2023-aug-16) *(bundled with plugin)*.
+
+# Platform Support
+
+## Supported:
+
+- Windows (Standalone x86 & x64)
+- Universal Windows Platform (x64)
+- [Android](/docs/android/README_Android.md) [^1]
+- [iOS](/docs/iOS/README_iOS.md) [^1]
+- Linux (Officially supported 18.04 LTS) [^1]
```

Comment:
We do not have a Linux readme document. We have a set of documents that discuss setting up a Linux environment for contributors. Something that should be changed, but not something within the scope of this PR.

Reply:
Scratch that - semantics - but yes I'll change that to link to the Ubuntu setup doc. We still should make a `README_Linux.md` document, since we have one for nearly all the other platforms.

Reply:
...aaaaaand done.